src/simple_http.rs

```diff
@@ -62,12 +63,12 @@ impl Default for SimpleHttpTransport {
 }
 
 impl SimpleHttpTransport {
-    /// Construct a new `SimpleHttpTransport` with default parameters
+    /// Constructs a new [`SimpleHttpTransport`] with default parameters.
```

Comment:
Imperative mood is idiomatic, AFAIK.

Reply:
I certainly hope not, I've been changing everything to third person for the last 10 months in rust-bitcoin :)

Reply:
Oh my...

... I just checked stdlib ... and indeed ...

... I've been living a lie.

Reply:
:)